src/components/borderedRow/BorderedRow.tsx

```diff
@@ -0,0 +1,18 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {borderedRow} from './styles/BorderedRow.scss';
+
+export class BorderedRow extends React.PureComponent<React.HTMLAttributes<HTMLDivElement>> {
+    static defaultClassName = classNames(borderedRow, 'bg-light-grey border-medium-grey mod-border-top mod-border-bottom');
+
+    render() {
+        return (
+            <div

```

Comment:
Since it's a BorderedRow I expected a TR, maybe you should rename for `Bordered` to avoid confusion